microbench/src/main/java/io/netty5/handler/codec/http/QueryStringEncoderBenchmark.java

```diff
@@ -86,10 +86,6 @@ private static String encode(String s) {
     }
 
     private static String repeat(String s, int num) {
-        StringBuilder sb = new StringBuilder(num * s.length());
-        for (int i = 0; i < num; i++) {
-            sb.append(s);
-        }
-        return sb.toString();
+        return s.repeat(Math.max(0, num));
```

Comment:
We can inline this method and won't need the `Math.max` call as `num` are then constants.